Core/ALife.Core/ALife.Core.csproj

```diff
@@ -15,7 +15,46 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Include="Utility\Maths\ExtraMaths.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ExtraMaths.tt</DependentUpon>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="Utility\Maths\ExtraMaths.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>ExtraMaths.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Utility\Maths\ExtraMath.cs">
```

Comment:
This whole section here has some confusion regarding "Math" vs "Maths"
I think, for our project, we should unify on "Math"

Reply:
We can def move back to Math rather than Maths if you'd prefer, it'd only take a couple of mins. This was mostly done to differentiate the old and new ExtraMath files while I was working on things

Reply:
As per offline discussion, namespace is Maths, files use Math